requirements.txt

```diff
@@ -52,7 +52,7 @@ requests==2.31.0
 robotframework==4.1.2
 s3transfer==0.3.7
 six==1.16.0
-tenacity==8.0.1
+tenacity>=8.0.1
```

Comment:
It's a good idea to write why in a separate commit message.

Reply:
fixed